_oasis

```diff
@@ -56,7 +56,7 @@ Library xenopsd
     Xenops_server_plugin,
     Xenops_server_skeleton,
     Scheduler,
-    Path,
+    Xpath,
```

Comment:
Thanks for the info, updated the names of the modules. I'm going to double-check in planex-buildenv whether this breaks the other components.